app/server/api/credit_transfer_api.py

```diff
@@ -48,17 +48,17 @@ def get(self, credit_transfer_id):
                 return make_response(jsonify({'message': 'Credit transfer not found'})), 404
 
             if AccessControl.has_sufficient_tier(g.user.roles, 'ADMIN', 'admin'):
-                transfer_list = credit_transfers_schema.dump([credit_transfer]).data
+                result = credit_transfer_schema.dump(credit_transfer).data
             elif AccessControl.has_any_tier(g.user.roles, 'ADMIN'):
-                transfer_list = view_credit_transfers_schema.dump([credit_transfer]).data
+                result = view_credit_transfer_schema.dump(credit_transfer).data
 
             transfer_stats = []
 
             response_object = {
                 'status': 'success',
                 'message': 'Successfully Loaded.',
                 'data': {
-                    'credit_transfer': transfer_list,
+                    'credit_transfer': result,
                     'transfer_stats': transfer_stats
```

Comment:
Individual credit transfer was returning a list with a single item. Changed format to follow the expected payload structure as per rest of app